workflow/automation/execution_scripts/query_mgmt_db.py

```diff
@@ -148,55 +151,53 @@ def get_all_entries(db, run_name, query_mode):
     return status
 
 
+# noinspection SqlResolve,SqlNoDataSourceInspection
```

Comment:
Is this meant to be a comment?

Reply:
Oops, pycharm control codes that snuck in